src/platform/user/profile/vet360/tests/e2e/routes.json

```diff
@@ -0,0 +1,173 @@
+[
+  {
+    "path": "/v0/feature_toggles",
+    "verb": "get",
+    "value": {
+      "data": {
+        "features": [
+          {
+            "name": "vaProfileAddressValidation",
+            "value": true
+          }
+        ]
+      }
+    }
+  },
+  {
+    "path": "/v0/address/countries",
+    "verb": "get",
+    "value": {
+      "data": {
+        "attributes": {
+          "countries": ["USA"]
+        }
+      }
+    }
+  },
+  {
+    "path": "/v0/address/states",
+    "verb": "get",
+    "value": {
+      "data": {
+        "attributes": {
+          "states": ["Alabama", "Kentucky"]
+        }
+      }
+    }
+  },
+  {
+    "path": "/v0/profile/full_name",
+    "verb": "get",
+    "value": {
+      "data": {
+        "attributes": {
+          "first": "First",
+          "middle": "Middle",
+          "last": "Last"
+        }
+      }
+    }
+  },
+  {
+    "path": "/v0/profile/personal_information",
+    "verb": "get",
+    "value": {
+      "data": {
+        "attributes": {
+          "birthDate": "1972-03-21",
+          "gender": "M"
+        }
+      }
+    }
+  },
+  {
+    "path": "/v0/profile/service_history",
+    "verb": "get",
+    "value": {
+      "data": {
+        "attributes": {
+          "serviceHistory": [
+            {
+              "branchOfService": "Army",
+              "beginDate": "1951-09-04",
+              "endDate": "1957-01-04"
+            }
+          ]
+        }
+      }
+    }
+  },
+  {
+    "path": "/v0/profile/email",
+    "verb": "put",
+    "value": {
+      "data": {
+        "attributes": {
+          "transaction_id": "transaction_1",
+          "transaction_status": "RECEIVED"
+        }
+      }
+    }
+  },
+  {
+    "path": "/v0/profile/status/transaction_1",
+    "verb": "put",
+    "value": {
+      "data": {
+        "attributes": {
+          "transaction_id": "transaction_1",
+          "transaction_status": "COMPLETED_SUCCESS"
+        }
+      }
+    }
+  },
```

Comment:
...to this line.

Reply:
It seems like some / most are necessary - I removed them and the tests started failing